docs/api/sim-card-usage.md

```diff
@@ -64,32 +68,33 @@ Data usage for a given **month** in a given **year** can be accessed by using th
 ```
 
 !!! tip
-    Keep in mind that the response is given as a list of usages and not returned as a regular top level JSON object.
-
+Keep in mind that the response is given as a list of usages and not returned as a regular top level JSON object.
 
 ## Message usage for a SIM card
+
 Message usage for a given **month** in a given **year** can be accessed by using the following URL:
 
 `GET /simcards/{icc}/usage/msg/{year}/{month}`
 
 **Response structure and types**
 
-| Field           | Type              | Description                                                     |
-| --------------- | ----------------- | --------------------------------------------------------------- |
-| icc             | String            | The unique identifier for the SIM card generating the usage.    |
-| year            | Number            | Year is given in the ISO 8601 format for example 2019.          |
-| month           | Number            | Month is represented as a number for example 'april = 4'        |
-| usageDate       | ISO 8601 DateTime | In UTC time.                                                    |
-| caller          | String            | Regular string and not necessarily a valid number.              |
-| called          | String            | E164 MSISDN, e.g. *+4593709603*                                 |
-| fromCountryCode | String            | Alpha-2 code                                                    |
-| toCountryCode   | String            | Alpha-2 code                                                    |
-| direction       | [Directions](/general-information/data-types#directions)            |               |
+| Field           | Type                                                     | Description                                                  |
+|-----------------|----------------------------------------------------------|--------------------------------------------------------------|
+| icc             | String                                                   | The unique identifier for the SIM card generating the usage. |
```

Comment:
Suggestion to add a `<br>` in description text, to create multiline text. With current text length I get a scroll bar at the bottom.